docs/quick_tutorial/authentication/setup.py

```diff
@@ -2,7 +2,7 @@
 
 requires = [
     'bcrypt',
-    'pyramid',
+    'pyramid[testing]',
```

Comment:
same comment as https://github.com/Pylons/pyramid/pull/3376#discussion_r223190034

Reply:
I standardized setup.py throughout the Quick Tutorial, and used the GitHub UI to clean it up. Resolving conversation.